### DISCUSSION TITLE
执行taro build --type harmony --watch提示，找不到插件依赖 "@tarojs/plugin-framework-react“，安装又提示找不到

### DISCUSSION THREAD

**Discussion by paihua514** [NervJS/taro#14928] [鸿蒙 && OpenHarmony 适配小组]
执行taro build --type harmony --watch提示如下：
![微信图片_20231204180019](https://github.com/NervJS/taro/assets/39114242/21ba4e61-88fb-44d3-ae53-ce0812a03de9)

执行npm i @tarojs/plugin-framework-react 提示如下：
![微信图片_20231204180416](https://github.com/NervJS/taro/assets/39114242/c6bd935f-6208-4c2e-aed8-8db7415f9a61)

**Comment by TheKonka** [comment]
> 执行npm i @tarojs/plugin-framework-react 提示如下：


你安装的不是 @tarojs/plugin-framework-harmony 吗？

**Reply by paihua514** [reply]
谢谢，这个问题已解决，是版本的问题，3.4.0改为3.6.0就可以了。但是执行taro build --type harmony --watch时，出错：ModuleParseError: Module parse failed: Duplicate export 'default' (9:7)。这是怎么回事呀。如下图
![微信图片_20231205100539](https://github.com/NervJS/taro/assets/39114242/b76a47b9-59e0-4a49-ae25-1ecfcf102356)
